tests/experimental.libsonnet

```diff
@@ -107,6 +108,7 @@ local volumes = import 'templates/volumes.libsonnet';
 
                 exit_code=$?
                 test $exit_code = 0 && break || sleep 30;
+                set -e
```

Comment:
Should this go up one line? Otherwise when the loop breaks, we'll still have `+e` set

Reply:
Great call. I just put it around the gcloud command.

Reply:
It should go after `exit_code=$?`, otherwise exit_code will always be 0 because `set -e` will alway work:

```
wcromar-macbookpro2:~ wcromar$ set +e
wcromar-macbookpro2:~ wcromar$ fail
-bash: fail: command not found
wcromar-macbookpro2:~ wcromar$ set -e
wcromar-macbookpro2:~ wcromar$ echo $?
0
wcromar-macbookpro2:~ wcromar$ 
```

Reply:
omg. I swear I thought of this the first time around :P Done.